feature/gnoi/system/tests/per_component_reboot_test/per_component_reboot_test.go

```diff
@@ -215,3 +216,80 @@ func TestLinecardReboot(t *testing.T) {
 	helpers.ValidateOperStatusUPIntfs(t, dut, intfsOperStatusUPBeforeReboot, 10*time.Minute)
 	// TODO: Check the line card uptime has been reset.
 }
+
+
+//Reboot the fabric component on the DUT.
+func TestFabricReboot(t *testing.T) {
+	dut := ondatra.DUT(t, "dut")
+	if deviations.GNOIFabricComponentRebootUnsupported(dut) {
+		t.Skipf("Skipping test due to deviation deviation_gnoi_fabric_component_reboot_unsupported")
+	}
+
+	const fabricBootTime = 10 * time.Minute
+	fabrics := components.FindComponentsByType(t, dut, fabricType)
+	t.Logf("Found fabric components: %v", fabrics)
+
+	t.Logf("Find a removable fabric component to reboot.")
+	var removableFaric string
+	for _, fabric := range fabrics {
+		t.Logf("Check if %s is removable", fabric)
+		if got := gnmi.Lookup(t, dut, gnmi.OC().Component(fabric).Removable().State()).IsPresent(); !got {
+			t.Logf("Detected non-removable fabric component: %v", fabric)
```

Comment:
Did you also want this logged when the value is present but false? The current code does not do that.

Reply:
Added the log when removeable value is false.